src/cpp/CMakeLists.txt

```diff
@@ -191,9 +191,6 @@ endif()
 set_target_properties(${TARGET_NAME} PROPERTIES
     EXPORT_NAME genai
     INTERPROCEDURAL_OPTIMIZATION_RELEASE ${ENABLE_LTO}
-    ARCHIVE_OUTPUT_DIRECTORY "$<1:${CMAKE_BINARY_DIR}/openvino_genai/>"
-    LIBRARY_OUTPUT_DIRECTORY "$<1:${CMAKE_BINARY_DIR}/openvino_genai/>"
-    RUNTIME_OUTPUT_DIRECTORY "$<1:${CMAKE_BINARY_DIR}/openvino_genai/>"
```

Comment:
Why is it removed? Should it use new vars instead?

Reply:
yeap. It uses global variables instead